modules/shared/src/main/scala/org/tessellation/currency/schema/FeeTransaction.scala

```diff
@@ -0,0 +1,22 @@
+package org.tessellation.currency.schema
+
+import org.tessellation.ext.derevo.ordering
+import org.tessellation.schema.address.Address
+import org.tessellation.schema.balance.Amount
+import org.tessellation.schema.transaction.{TransactionOrdinal, TransactionReference, TransactionSalt}
+
+import derevo.cats.{order, show}
+import derevo.circe.magnolia.{decoder, encoder}
+import derevo.derive
+
+@derive(decoder, encoder, order, ordering, show)
+case class FeeTransaction(
+  source: Address,
```

Comment:
@ryle-goehausen will this transaction always be signed by the owner? I wonder if we can maybe ditch the source field and just derive it from the signer id.

Reply:
I think it would be better to leave this in here as it was in original docs.